<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>326</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>326</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Batch Render Folder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="qtServerLabel">
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qtServerEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qtConnectBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>23</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="qtLogText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="qtPreviewRdo">
       <property name="text">
        <string>Preview</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="qtRecordRdo">
       <property name="text">
        <string>Record</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="qtStartBtn">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
